pkgs/by-name/hy/hyprland-activewindow/package.nix

```diff
@@ -2,16 +2,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "hyprland-activewindow";
-  version = "1.0.2";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "FieldOfClay";
     repo = "hyprland-activewindow";
     rev = "v${version}";
-    hash = "sha256-kF2dNb9hiC6RcL2XG8k18da5he94Jpv3v+HdfHbeW3E=";
+    hash = "sha256-HJrzAkv2vwS5q58gMmQylKrPNUTfcsvZ6JdAgZNqGxU=";
   };
 
-  cargoHash = "sha256-YCzAfVLKDECGG/1fs3vyVB0oglxLFSE+2cnmLc7RoEo=";
+  cargoPatches = [ ./fix-out-of-date-cargo-lock.patch ];
```

Comment:
```suggestion
```
Looks like a new tag was made upstream so the patch shouldn't be needed anymore.

Should be able to update the hashes with `lib.fakeHash`.

Reply:
Yes indeed! The PR was merged yesterday, but I was out of town. I'm committing the update as we speak.